README.md

```diff
@@ -87,39 +101,99 @@ The VTS is a metric designed to help stakers evaluate the performance and reliab
 The VTS is displayed in the Nimiq Wallet, allowing stakers to compare validators and select the one that best meets their needs.
 
 - [Read the docs](https://nimiq.com/developers/learn/validator-trustscore)
-- [See implementation](./packages/nimiq-validators-trustscore/)
+- Checkout the [pnpm package](./packages/nimiq-validator-trustscore/)
 
 ## Validators API
 
 The Validators API provides endpoints to retrieve validator information for integration with tools, dashboards, and other applications.
 | Endpoint | Description |
 | -------------------------------------------------------------------------------- | ------------------------------------------------------------------------------- |
 | [/api/v1/validators](https://validators-api-mainnet.pages.dev/api/v1/validators) | Retrieves the validator list. See [query params](./server/utils/schemas.ts#L54) |
+| [/api/v1/validators/validator_address](https://validators-api-mainnet.pages.dev/api/v1/validators/validator_address) | Retrieves the validator information |
```

Comment:
Not very intuitive, maybe use
```suggestion
| [/api/v1/validators/:validator_address](https://validators-api-mainnet.pages.dev/api/v1/validators/NQ7700000000000000000000000000000001) | Retrieves the validator information |
```

And the address with spaces is not supported, this should be documented too